test/infrastructure/docker/api/v1beta1/devmachine_types.go

```diff
@@ -66,6 +66,119 @@ const (
+
+	// DevMachineInMemoryVirtualMachineWaitingForClusterInfrastructureV1Beta2Reason documents a fake VirtualMachine for a DevMachine's with in memory backend waiting for the cluster
+	// infrastructure to be ready.
+	DevMachineInMemoryVirtualMachineWaitingForClusterInfrastructureV1Beta2Reason = clusterv1.WaitingForClusterInfrastructureV1Beta2Reason
+
+	// DevMachineInMemoryVirtualMachineWaitingControlPlaneInitializedV1Beta2Reason documents a fake VirtualMachine for a DevMachine's with in memory backend waiting
+	// for the control plane to be initialized.
+	DevMachineInMemoryVirtualMachineWaitingControlPlaneInitializedV1Beta2Reason = clusterv1.WaitingControlPlaneInitializedV1Beta2Reason
+
+	// DevMachineInMemoryVirtualMachineWaitingForBootstrapDataV1Beta2Reason documents a fake VirtualMachine for a DevMachine's with in memory backend waiting for the bootstrap
+	// data to be ready.
+	DevMachineInMemoryVirtualMachineWaitingForBootstrapDataV1Beta2Reason = clusterv1.WaitingForBootstrapData1Beta2DataReason
+
+	// DevMachineInMemoryVirtualMachineWaitingForStartupTimeoutV1Beta2Reason documents the infrastructure for a fake VirtualMachine for a DevMachine's with in memory backend provisioning.
+	DevMachineInMemoryVirtualMachineWaitingForStartupTimeoutV1Beta2Reason = "WaitingForStartupTimeout"
+
+	// DevMachineInMemoryVirtualMachineProvisionedV1Beta2Reason documents a fake VirtualMachine for a DevMachine's with in memory backend being provisioned.
+	DevMachineInMemoryVirtualMachineProvisionedV1Beta2Reason = clusterv1.ProvisionedV1Beta2Reason
+
+	// DevMachineInMemoryVirtualMachineInternalErrorV1Beta2Reason surfaces unexpected error when reconciling a fake VirtualMachine for a DevMachine's with in memory backend.
+	DevMachineInMemoryVirtualMachineInternalErrorV1Beta2Reason = clusterv1.InternalErrorV1Beta2Reason
+)
+
+// NodeProvisioned condition and corresponding reasons that will be used in v1Beta2 API version for a DevMachine's with in memory backend.
+const (
+	// DevMachineInMemoryNodeProvisionedV1Beta2Condition documents the status of the provisioning of the node hosted on the InMemoryMachine.
+	DevMachineInMemoryNodeProvisionedV1Beta2Condition string = "NodeProvisioned"
+
+	// DevMachineInMemoryNodeWaitingForVirtualMachineProvisionedV1Beta2Reason documents a fake Node for a DevMachine's with in memory backend waiting for the VirtualMachine
+	// to be provisioned.
+	DevMachineInMemoryNodeWaitingForVirtualMachineProvisionedV1Beta2Reason = "WaitingForVirtualMachineProvisioned"
+
+	// DevMachineInMemoryNodeWaitingForStartupTimeoutV1Beta2Reason documents the fake Node for a DevMachine's with in memory backend provisioning.
+	DevMachineInMemoryNodeWaitingForStartupTimeoutV1Beta2Reason = "WaitingForStartupTimeout"
+
+	// DevMachineInMemoryNodeProvisionedV1Beta2DataReason documents a fake Node for a DevMachine's with in memory backend being provisioned.
+	DevMachineInMemoryNodeProvisionedV1Beta2Reason = clusterv1.ProvisionedV1Beta2Reason
```

Comment:
```suggestion
	// DevMachineInMemoryNodeProvisionedV1Beta2Reason documents a fake Node for a DevMachine's with in memory backend being provisioned.
	DevMachineInMemoryNodeProvisionedV1Beta2Reason = clusterv1.ProvisionedV1Beta2Reason
```

\+ Same comment as in l.110